scripts/proto/lbann/proto.py

```diff
@@ -170,9 +170,14 @@ def export_proto(self):
                     setattr(layer_message, field_name, v)
         return proto
 
+    # Method for returning the names of all fields.
+    def get_field_names(self):
+        return field_names
+
     # Create sub-class.
     return type(type_name, (Layer,),
-                {'__init__': __init__, 'export_proto': export_proto})
+                {'__init__': __init__, 'export_proto': export_proto,
```

Comment:
I reorganized `proto.py` recently, so I think there will be some minor merge conflicts in this file. It should be a straightforward fix, but let me know if you need help.

Reply:
Fixed in oyamay/lbann@5ce57a5.